src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitStorage {
+    public static final List<Fruit> fruits = new ArrayList<>();
```

Comment:
You can use the Map, where the key is the name of the Fruit and the value quantity
or leave your choice
```suggestion
    public static final Map<String, Integer> fruits = new HashMap<>();
```